### ISSUE TITLE
filesystem: failed to execute df command in OpenWrt (with BusyBox)

### ISSUE THREAD

**Issue by musashino205** [mackerelio/mackerel-agent#568]
In OpenWrt, mackerel-agent fails to execute ```df``` command and prints a lot of warnings.

log:
```
2019/06/14 10:56:01 WARNING <util.filesystem> 'df -Pkl' command exited with a non-zero status: 1: "df: unrecognized option: l\nBusyBox v1.30.0 () multi-call binary.\n\nUsage: df [-PkmhT] [FILESYSTEM]...\n\nPrint filesystem usage statistics\n\n\t-P\tPOSIX output format\n\t-k\t1024-byte blocks (default)\n\t-m\t1M-byte blocks\n\t-h\tHuman readable (e.g. 1K 243M 2G)\n\t-T\tPrint filesystem type\n"
...
```

BusyBox df in OpenWrt outputs ```df: unrecognized option: l``` as a warning, so it seems that it has slipped through the check in ```func init()```.

**Comment by astj** [comment]
Ref: https://github.com/mackerelio/mackerel-agent/blob/master/util/filesystem.go#L56

**Comment by astj** [comment]
@musashino205 Hi, thank you for reporting.

First, we officially do not support neither OpenWrt or BusyBox. Our official support targets are described in [help docs](https://mackerel.io/docs/entry/overview).

And I made a simple fix attempt at #570. Can you try the branch and tell us the result?

**Comment by musashino205** [comment]
@astj Thank you for your information.
Certainly I did not check the documentation for the supported environments. Sorry.

> And I made a simple fix attempt at #570. Can you try the branch and tell us the result?

OK, I'll try it.

**Comment by musashino205** [comment]
Tried, it works very well. Thank you!

**Comment by astj** [comment]
Thank you for testing and reporting! I'd like to get #570 reviewed by our team members. Please wait a bit until it merged into master.

**Comment by astj** [comment]
Merged #570 into master, therefore I'd like to close this issue.
